queries/racket/highlights.scm

```diff
@@ -74,6 +74,12 @@
    ))
 
+(list
+  .
+  (symbol) @keyword.coroutine
+  (#any-of? @keyword.coroutine
+  "for/async" "for*/async"))
```

Comment:
I'm sorry to comment this again. But `for/async`, `for*/async` are not coroutine equivalent in Racket.
There are no coroutines in Racket. `engine` like coroutine but it can be preempted by a external
trigger. Another way is to use continuation.
I think we should cancel racket support here.

Reply:
No don't be sorry, I appreciate the feedback! I'll remove the changes to racket now

Reply:
Btw, is this relevant? https://docs.racket-lang.org/reference/async-channel.html

Reply:
No. They are different with coroutine.
`async-channel` is for synchronize between threads.
`future` is for parallel, not concurrency.

People can implements coroutines in continuation, and give its name whatever they want.
It should not exceed 30 lines of code. That's why Racket say it has coroutine support.

Reply:
ty!